src/typescript/frontend/src/components/StatsBar.tsx

```diff
@@ -1,51 +1,324 @@
+            </span>
+          </p>
+        </div>
+        {/* 24 hr pair */}
+        <div className="stat">
+          <span className="font-roboto-mono text-base font-light uppercase text-neutral-400">
+            24h volume ({marketData.data?.pairData.quoteAsset || "-"})
+          </span>
+          <p className="font-roboto-mono font-light">
+            <span className="text-white">
+              {marketData.data?.pairData.quoteVolume || "-"}
+            </span>
           </p>
         </div>
       </div>
-      <div className="my-auto">
-        <WalletSelector />
+      <div className="my-auto hidden md:block">
+        <SocialMediaIcons />
       </div>
     </div>
   );
 }
+
+// MOCK FUNCTIONS + DATA
+export const useMarketData = (market: string): UseQueryResult<MarketStats> => {
+  return useQuery(
+    ["marketStats", market],
+    async () => {
+      // MOCK API CALL
+      const baseAsset = market.split("-")[0];
+      const quoteAsset = market.split("-")[1];
+      function timeout(ms: number | undefined) {
+        return new Promise((resolve) => setTimeout(resolve, ms));
+      }
+      await timeout(1000);
+      const {
+        lastPrice,
+        change24h,
+        high24h,
+        low24h,
+        pairData,
+        lastPriceChange,
+        change24hPercent,
+      } = STATS_BAR_MOCK_DATA;
+      // END MOCK API CALL
+      return {
+        lastPrice: formatDecimal(lastPrice),
+        lastPriceChange: formatDecimalWithPlusMinus(lastPriceChange, 4),
+        change24h: formatDecimalWithPlusMinus(change24h, 4),
+        change24hPercent: formatDecimalWithPlusMinus(change24hPercent),
+        high24h: formatDecimal(high24h, 4),
+        low24h: formatDecimal(low24h, 4),
+        pairData: {
+          baseAsset: baseAsset,
+          quoteAsset: quoteAsset,
+          baseAssetIcon: pairData.baseAssetIcon,
+          quoteAssetIcon: pairData.quoteAssetIcon,
+          baseVolume: formatDecimal(pairData.baseVolume),
+          quoteVolume: formatDecimal(pairData.quoteVolume),
+        },
+      } as MarketStats;
+    },
+    { keepPreviousData: true, refetchOnWindowFocus: false }
+  );
+};
+
+const STATS_BAR_MOCK_DATA: MarketStats = {
+  lastPrice: "10.17",
+  lastPriceChange: "10.1738",
+  change24h: "-.9305",
+  change24hPercent: "-8.38",
+  high24h: "11.1681",
+  low24h: "9.85",
+  pairData: {
+    // asset names here don't matter for testing purposes
+    baseAsset: "doesn't",
+    quoteAsset: "matter",
+    baseAssetIcon: "/tokenImages/APT.png",
+    quoteAssetIcon: "/tokenImages/USD.png",
+    baseVolume: "6531688.77",
+    quoteVolume: "68026950.84",
+  },
+};
+
+// UTIL FUNCTIONS
+
+// format number to dollar
+const formatDecimal = (num: string, digits = 2): string => {
+  const roundedNum = Number(num).toFixed(digits);
+  const parts = roundedNum.split(".");
+  parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+  return parts.join(".");
+};
```

Comment:
You can just use `Intl.NumberFormat()` if you need to format a dollar amount, or `Number.prototype.toLocaleString()` if you need to set the number of decimals (as you have to here, since the quote currency isn't guaranteed to be a USD stablecoin).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString